tests/test_vqe.py

```diff
@@ -1253,7 +1253,7 @@ def circuit():
 
             res_expected.append(circuit())
 
-        res_expected = np.sum(c * r for c, r in zip(coeffs, res_expected))
+        res_expected = np.sum([c * r for c, r in zip(coeffs, res_expected)])
```

Comment:
I saw a deprecation warning for sums of generators...

Reply:
:+1: